test/base/SpecBase.scala

```diff
@@ -76,6 +92,19 @@ trait SpecBase
     Future.successful(Right(()))
   )
 
+  val badSessionService: SessionService = mock[SessionService]
+
+  when(badSessionService.readUserAnswers(any[InternalId])) thenReturn
```

Comment:
why are we doing this here? :)

Reply:
Because it's used in a lot of different specs - is there a better way?

Reply:
I think my instinct would be to not put mocks in the base, because its hard to tell in the individual test where the mocking is happening. Could be middle ground, e.g. a function that has all the mocks in and you can call that from each spec?

Reply:
this is done for the sessionService mock above it though - do we removed that too?

Reply:
I like this idea - much better